src/data/zwave.ts

```diff
@@ -0,0 +1,19 @@
+import { HomeAssistant } from "../types";
+
+export interface ZWaveNetworkStatus {
+  state: number;
+  state_str: string;
```

Comment:
Didn't we change this ?

Reply:
Yup, `state_str` isn't used anymore, so I'll remove it